docs/commerce/checkout/customize_checkout.md

```diff
@@ -192,6 +192,58 @@ You should be able to see a one page checkout applied after you add products to
 
 ![One page checkout](img/single_page_checkout.png "One page checkout")
 
+## Define custom checkout workflow
+
+This example shows how to create custom checkout `new_workflow` and its strategy.
```

Comment:
```suggestion
This example shows how to create strategy for a custom checkout `new_workflow`.
```

In fact `new_workflow` it self is not defined in this example. I guess `new Workflow('new_workflow')` needs `new_workflow` to be defined to work.

In 4.6.0-beta.1, this how `ibexa_virtual_products` workflow works.
Its strategy return it: https://github.com/ibexa/checkout/blob/v4.6.0-beta1/src/lib/Workflow/Strategy/VirtualProductsWorkflow.php#L22
it's defined in YAML config: https://github.com/ibexa/checkout/blob/v4.6.0-beta1/src/bundle/Resources/config/workflow.yaml#L3

(In main branch, it moved to conditional step. `ibexa_virtual_products` has been merged into `ibexa_checkout` and conditional step make it take the `select_billing_address` step when virtual and `select_billing_address` otherwise)